python/fastquant/fastquant_old.py

```diff
@@ -650,3 +651,74 @@ def get_bt_news_sentiment(keyword, page_nums=None):
         ] = round(sum(v) / float(len(v)), 3)
 
     return date_sentiment
+
+
+def get_twitter_sentiment(stock_code, twitter_auth, start_date, twitter_accounts=None):
```

Comment:
fastquant was recently refactored and `fastquant_old.py` will be deprecated soon.
Could you please migrate the changes into the appropriate module below?

https://github.com/enzoampil/fastquant/blob/master/python/fastquant/data/web/twitter.py

Reply:
> > @vincevertulfo please add a test on using this one to the sentiment strat. thanks
> 
> Hi @rafmacalaba, is this the same as the test_twitter.py that @jpdeleon mentioned above?

no, it's not. see test_strategies.py it has get_bt_news, make sure it works like that one.